tasks.py

```diff
@@ -187,7 +187,7 @@ def assets_build(progress=False):
     if progress:
         cmd += ' --progress'
     lrun(cmd.format('webpack.config.prod'), pty=True)
-    lrun(cmd.format('webpack.widgets.config'), pty=True)
+    lrun(cmd.format('webpack.widgets.config.prod'), pty=True)
```

Comment:
In order to be consistent, files should be named: `webpack.config.*.js`. In this case, `webpack.config.widgets.prod.js`.